WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -396,16 +396,24 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField
 
 - (void)configureSections
 {
-    self.sections = @[
-                      @(PostSettingsSectionTaxonomy),
-                      @(PostSettingsSectionMeta),
-                      @(PostSettingsSectionFormat),
-                      @(PostSettingsSectionFeaturedImage),
-                      @(PostSettingsSectionStickyPost),
-                      @(PostSettingsSectionShare),
-                      @(PostSettingsSectionGeolocation),
-                      @(PostSettingsSectionMoreOptions)
-                      ];
+    NSNumber *stickyPostSection = @(PostSettingsSectionStickyPost);
+    NSMutableArray *sections = [@[@(PostSettingsSectionTaxonomy),
+                                  @(PostSettingsSectionMeta),
+                                  @(PostSettingsSectionFormat),
+                                  @(PostSettingsSectionFeaturedImage),
+                                  stickyPostSection,
+                                  @(PostSettingsSectionShare),
+                                  @(PostSettingsSectionGeolocation),
+                                  @(PostSettingsSectionMoreOptions)]
+                                mutableCopy];
+    
+    // Self-hosted non-Jetpack blogs have no capabilities,
+    // so we'll remove the Sticky Post section.
+    //
+    if (self.post.blog.capabilities == nil) {
```

Comment:
Since `capabilities` is something we get from the REST API, would it be better to check `![self.post.blog supportsRestApi]`?

Reply:
Yep! It makes sense! I used the same logic I found into the AztecPostViewController when a context is created.